tests/InputFiles/NewtonianEuler/RiemannProblem1D.yaml

```diff
@@ -63,6 +63,39 @@ EventsAndTriggers:
         Specified:
           Values: [10]
     - - Completion
+  # - - TimeCompares:
```

Comment:
Question: is it reasonable to specify a final time rather than number of steps, even it's quite a bit shorter than the final time of the actual problem? Just so that people only need to specify the final time they want. I guess the nice thing about specifying slabs is we know in the GTS case that no matter what adaptivity does we will have a specific number of steps... :shrug:

Reply:
Turns out that at the low resolution of the test the whole evolution to 0.2 only takes 60 steps, which is probably fast enough to just run.  I left the LTS one with a fixed slab number because that's a reasonable condition if LTS is being used without varying the slab size.